src/nodes/display/PassNode.js

```diff
@@ -680,6 +724,83 @@ class PassNode extends TempNode {
 
 		this.renderTarget.setSize( effectiveWidth, effectiveHeight );
 
+		if ( this._customScissor !== null ) this.renderTarget.scissor.copy( this._customScissor );
+		if ( this._customViewport !== null ) this.renderTarget.viewport.copy( this._customViewport );
+
+	}
+
+	/**
+	 * By default, the scissor rectangle is kept in sync with the pass's
+	 * resolution. This method allows to define a custom scissor rectangle.
```

Comment:
>By default, the scissor rectangle is kept in sync with the pass's resolution.

It is not clear to me what you mean by that sentence. Can you explain further?

Reply:
The width and height arguments passed to `RenderTarget.setSize()` define the values of `viewport` and `scissor`.

https://github.com/mrdoob/three.js/blob/300b0fa760832d62391007a8cf1aab0c0b58c11f/src/core/RenderTarget.js#L305-L306

That is not always wanted. When using a custom viewport and scissor, `PassNode` makes sure they are not overwritten.